common_app/io.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <stdint.h>
+
+#include "ux.h"
+#include "os_io_seproxyhal.h"
+
+#include "types.h"
+#include "buffer.h"
+#include "macros.h"
+
+#ifdef HAVE_BAGL
+WEAK void io_seproxyhal_display(const bagl_element_t *element);
+#endif  // HAVE_BAGL
+
+/**
+ * IO callback called when an interrupt based channel has received
+ * data to be processed.
+ *
```

Comment:
Maybe we could add that this function has the responsibility to forward the received events to the UX lib.